src/main/webapp/WEB-INF/themes/root/app/approot.ftl

```diff
@@ -53,6 +53,7 @@
   Available under the
   <a href="http://creativecommons.org/licenses/by/2.0/deed.en">Creative Commons Attribution 2.0 Generic License.</a>
 </div>
+<h1>environment is: ${environment}</h1>
```

Comment:
This was just a POC. If you want to remove it also revert AppRootPage.java.  If you want to keep it, an `<h1>` is a little too loud IMO.